### PR TITLE
ZENKO-4103: bump Zenko 2.3.12

### PR DESCRIPTION
**Pull request by JBWatenbergScality**
**What does this PR do, and why do we need it?**

**Which issue does this PR fix?**

fixes #<ISSUE>

**Special notes for your reviewers**:

### CODE REVIEW

File: solution/deps.yaml [hunk 1]
```diff
@@ -90,7 +90,7 @@ zenko-operator:
 zenko-ui:
   sourceRegistry: registry.scality.com/zenko-ui
   image: zenko-ui
-  tag: 1.4.3
+  tag: 1.4.4
```
Comment (francoisferrand) [inline solution/deps.yaml:93]: should be in a separate commit or at least mentionned in commit message : so that it is easier to see in change log...

**Review by francoisferrand — APPROVED** [review]

**Review by ghost — APPROVED** [review]
